postgres/datadog_checks/postgres/postgres.py

```diff
@@ -1058,6 +1065,8 @@ def check(self, instance):
             db = self.get_connection(key, host, port, user, password, dbname, ssl, connect_fct, tags)
             version = self._get_version(key, db)
             self.log.debug("Running check against version %s" % version)
+            if tag_replication_role:
+                tags.extend(["replication_role:%s" % self._get_replication_role(key, db)])
```

Comment:
We've been moving to `.format()`, could we use that here:
`tags.extend(["replication_role:{}".format(self._get_replication_role(key,db))])`

Reply:
done